packages/preload/src/index.ts

```diff
@@ -1157,6 +1157,26 @@ export function initExposure(): void {
     },
   );
 
+  contextBridge.exposeInMainWorld('updatePodmanDesktop', async (): Promise<void> => {
+    return ipcInvoke('app:update');
+  });
+
+  contextBridge.exposeInMainWorld('podmanDesktopUpdateAvailable', async (): Promise<boolean> => {
+    return ipcInvoke('app:update-available');
+  });
+
+  contextBridge.exposeInMainWorld('podmanDesktopOpenReleaseNotes', async (version: string): Promise<void> => {
+    return ipcInvoke('app:open-release-notes', version);
```

Comment:
This is only used once (for 'current') by the UI, after having successfully received the podmanDesktopGetReleaseNotes. Why not just return the URL in the notesURL of podmanDesktopGetReleaseNotes (or add an extra property) and then the UI can just follow the link? This would remove some complexity/tests and need to expose this API.